### ISSUE TITLE
Massive fps drop with ai getting stuck in the sub

### ISSUE THREAD

**Issue by MaxwellBonnell** [Regalis11/Barotrauma#10369]
### Disclaimers

- [X] I have searched the issue tracker to check if the issue has already been reported.
- [ ] My issue happened while using mods.

### What happened?

Often when dragging npc's from outposts or abandoned ones down to the sub through the docking ladder when unable to reach the ladder to climb back out it will often cause a huge fps drop

### Reproduction steps

-Dock at an outpost.
-Drag a outpost npc (outpost security guards works most of the time) down to the sub. 
-Get them stuck trying to walk towards the docking ladder(closing them behind a door or just holding them with the heal interface).

FPS goes back to normal when the npc either dies or successfully returns to the outpost

### Bug prevalence

Happens every now and then

### Version

0.19.14.0

### -

Also happens in the unstable branch

### Which operating system did you encounter this bug on?

Windows

### Relevant error messages and crash reports

```shell
none
```

**Comment by Regalis11** [comment]
Thank you for the report (and sorry for taking so long for us to look into this issue)!

I gave this a test, but unfortunately couldn't get any kind of performance hit to occur. I tried the following:
- Stunned and dragged an outpost security officer into the sub and started a fight with them
- Did the same with a normal outpost NPC
- Teleported all the NPCs from the outpost to the sub, and started attacking them

None of this seemed to incur any measurable performance hit. So I think this requires some more specific circumstances to occur - perhaps some outpost or sub with the waypoints laid out in a specific way?
